cli/actions/setup.js

```diff
@@ -86,6 +86,9 @@ module.exports = (config, flags, args) => {
       commands = uniq(commands.concat(tempScripts));
     }
 
+    // This is the default test script added by 'npm init'.
+    const npmInitDefaultTestScript = 'echo \"Error: no test specified\" && exit 1'; // eslint-disable-line no-useless-escape
```

Comment:
I think eslint considers it a "useless escape" because the escape character needs to be escaped, i.e., this should be `'echo \\"Error...'`. could you write an end to end test (in `e2e_tests/tests`) to verify?